common/borders/borders-en.html

```diff
@@ -0,0 +1,125 @@
+---
+{
+    "title": "Borders",
+    "language": "en",
+    "altLangPage": "borders-fr.html",
+    "breadcrumbs":
+        [
+            {
+                "title": "GCWeb home",
+                "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html",
+            },
+        ],
+    "secondlevel": false,
+    "dateModified": "2023-06-20",
+    "share": "true",
+}
+---
+
+<p>
+	Borders are commonly used to provide visual separation between elements,
+	highlight specific content, or create decorative effects. By adjusting the
+	border properties, you can customize the appearance of the borders to match
+	your desired design.&nbsp;
+</p>
```

Comment:
Please remove this descriptive blurb, and instead bring to the top the list you created at the bottom of this page.